jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java

```diff
@@ -444,18 +445,14 @@ private static String printWebElement(WebElement element) {
         );
     };
 
-    public void actions(JFunc2<Actions, WebElement, Actions> actions) {
-        actions.execute(actionsClass(), get()).build().perform();
+    public void actions(JFunc2<Actions, WebElement, Actions> action) {
+        action.execute(actions.get(), get()).build().perform();
     }
-    public void actionsWitElement(JFunc2<Actions, WebElement, Actions> actions) {
-        actions.execute(actionsClass().moveToElement(get()), get()).build().perform();
-    }
-    private Actions actions = null;
-    private Actions actionsClass() {
-        if (actions == null)
-            actions = new Actions(driver());
-        return actions;
+    public void actionsWitElement(JFunc2<Actions, WebElement, Actions> action) {
+        action.execute(actions.get().moveToElement(get()), get()).build().perform();
     }
+    private Safe<Actions> actions = new Safe<>(() -> new Actions(driver()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4817150)